jerry.sh

```diff
@@ -1170,6 +1196,33 @@ read_manga_choice() {
     [ "$score_on_completion" = true ] && update_score "MANGA" "immediate"
 }
 
+create_room() {
+    local room_name=$(curl -s -X POST "https://backend.watchparty.me/createRoom" d '{"video":""}' | jq .name -r)
+    echo "$room_name" >"$data_dir/last_room_name.txt"
+    echo "$room_name"
+}
+
+add_video_to_room() {
+    # todo: don't print everything that's happening here, and
+    # specifically killing the process if it's running on port 12345 and 
+    # is websocat
```

Comment:
to avoid printing to stdout just redirect the output like this `>/dev/null`

for websocat running on port 12345 it's prob just easier to kill websocat alltogether, if it's already running